MovieTime/MovieTimeTests/MTMovieFeedViewModelTests.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  MTMovieFeedViewModelTests.swift
+//  MovieTimeTests
+//
+//  Created by Enrique Melgarejo on 09/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import XCTest
+@testable import MovieTime
+
+class MTMovieFeedViewModelTests: XCTestCase {
```

Comment:
your tests are not building, would you like to have some additional time to fix that?

Reply:
At the end of the project I made important changes to the Service initializer and did not change the tests.
Yes I would like. Testing is important, but often customers do not pay for it or it stay in background when planning a sprint. I decided not to do them to be able to make the delivery as planned. Who knows, next time I plan the test resolution including unit tests.

Reply:
I understand your point and we hear all the time complaints about clients that don't want "to pay for unit tests", but just to give you some taste about our culture: at ArcTouch we believe that unit tests are a developer tool, just like Xcode is one. Ideally you can't develop without the appropriate tools. We don't charge a client for unit tests, in the same way that we don't charge for using Xcode. And if they don't want to see us coding tests, we don't commit them lol

Reply:
Great culture! This shows how correct the information I've seen about ArchTouch is and makes me excited to be participating in the selection process!